proposals/0273-webengine-projection-mode.md

```diff
@@ -107,6 +136,10 @@ Independent of the app presentation type, the HMI will continue to provide syste
 
 With the current depth of this proposal, the HMI type should be used by 1st party OEM apps only. With future proposals and workshops the SDLC could open the HMI type to 3rd party by creating and defining proper driver distraction and user interface guidelines.
 
+The acceptance of this proposal means the feature is available for OEM/1st party apps and content ONLY.
+There is an inherent risk that OEMs could use the feature to bring in 3rd party apps and content without additional proposals
+as it will be impossible to enforce. If an OEM takes such an action, the SDLC will be forced to bring the issue up to the Board of Directors for possible consequences.

```

Comment:
```suggestion
The acceptance of this proposal means the feature is available for OEM/1st party apps and content ONLY. There is an inherent risk that OEMs could use the feature to bring in 3rd party apps and content without additional proposals as it will be impossible to enforce. If an OEM takes such an action, the SDLC will be forced to bring the issue up to the Board of Directors for possible consequences.
```